build/dependencies.props

```diff
@@ -11,9 +11,9 @@
     <RuntimeFrameworkVersion Condition="'$(TargetFramework)'=='netcoreapp2.0'">2.0.0-*</RuntimeFrameworkVersion>
     <RoslynVersion>2.3.0-beta1</RoslynVersion>
     <RoslynDevVersion>2.3.0-beta3-61731-06</RoslynDevVersion>
-    <StreamJsonRpcVersion>1.0.2-rc</StreamJsonRpcVersion>
+    <StreamJsonRpcVersion>1.1.73</StreamJsonRpcVersion>
     <TestSdkVersion>15.3.0-*</TestSdkVersion>
-    <VsShellVersion>15.0.26201</VsShellVersion>
```

Comment:
What VS release does this correspond to? I imagine it's 15.1 but can we find out for sure?